sdk/batch/arm-batch/package.json

```
@@ -43,14 +51,15 @@
     "esm/**/*.d.ts",
     "esm/**/*.d.ts.map",
     "src/**/*.ts",
+    "README.md",
     "rollup.config.js",
     "tsconfig.json"
   ],
   "scripts": {
     "build": "tsc && rollup -c rollup.config.js && npm run minify",
     "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/arm-batch.js.map'\" -o ./dist/arm-batch.min.js ./dist/arm-batch.js",
-    "prepack": "npm install && npm run build"
+    "prepack": "npm install && npm run build",
+    "test": "ts-mocha -p tsconfig.test.json test/*.spec.ts --timeout 100000"
```

I would suggest the following which can be used with the [automatic debugging configuration](https://code.visualstudio.com/updates/v1_45#_automatic-debug-configurations) for Node.js in Visual Studio Code.
```suggestion
    "test": "mocha -r esm --require ts-node/register test/*.spec.ts --timeout 1200000 --full-trace"
```